### ISSUE TITLE
Dynamic references to chapter within a chapter

### ISSUE THREAD

**Issue by pauldpickell** [ubc-geomatics-textbook/geomatics-textbook#88]
Throughout the textbook, some chapters may reference other chapters. These instances need to be found (ctrl+F "chapter") and then dynamically linked. For example, Chapter 11 references Chapter 12 and this should be linked like:

`These specifications will be discussed in greater detail in [Chapter 12](https://ubc-geomatics-textbook.github.io/geomatics-textbook/remote-sensing-systems.html), so for now we’ll focus on how spectral information can be useful.`
